numpy/core/src/multiarray/_multiarray_tests.c.src

```diff
@@ -848,6 +848,24 @@ get_buffer_info(PyObject *NPY_UNUSED(self), PyObject *args)
 #undef GET_PYBUF_FLAG
 
 
+static PyObject*
+get_c_wrapping_array(PyObject* NPY_UNUSED(self), PyObject* arg)
+{
+    int writeable, flags;
+    npy_intp zero = 0;
+
+    writeable = PyObject_IsTrue(arg);
+    if (error_converting(writeable)) {
+        return NULL;
+    }
+
+    flags = writeable ? NPY_ARRAY_WRITEABLE : 0;
+    /* Create an empty array (which points to a random place) */
+    return PyArray_NewFromDescr(&PyArray_Type, PyArray_DescrFromType(NPY_INTP),
+                                1, &zero, NULL, &zero, flags, NULL);
+}
+
+
```

Comment:
Needs a comment what is the purpose of this function, it is a bit obscure at first glance.